lib/template-assembler.js

```diff
@@ -6,6 +6,44 @@ const upath = require('upath');
 
 const partialRegex = /\{\{>\s*([_|\-|a-zA-Z0-9/]+)[^{]*?}}/g;
 const dynamicComponentRegex = /\{\{\s*?dynamicComponent\s*(?:'|")([_|\-|a-zA-Z0-9/]+)(?:'|").*?}}/g;
+const includeRegex = /{{2}>\s*([_|\-|a-zA-Z0-9/]+)[^{]*?}{2}/g;
+const packageMarker = 'external/';
```

Comment:
Maybe it's better to consolidate this name with  `externalTemplatesPath` in `stencil-bundle`?

Reply:
We removed `externalTemplatesPath`
If your comment is still valid please can you add more details:)